python/ray/tune/schedulers/pb2.py

```diff
@@ -0,0 +1,322 @@
+from copy import deepcopy
+import logging
+from typing import Dict, Optional
+
+import numpy as np
+import pandas as pd
+import GPy
+
+from ray.tune.schedulers import PopulationBasedTraining
+from ray.tune.schedulers.pb2_utils import normalize, optimize_acq, \
+    select_length, \
+    standardize, TV_SquaredExp
+
+logger = logging.getLogger(__name__)
+
+
+## select config, given X and y.
```

Comment:
Can you remove this comment and add a docstring to this function instead, describing what it does and also what the args are.

Reply:
Done.